### ISSUE TITLE
[WIKI] Table structure update

### ISSUE THREAD

**Issue by BarbzYHOOL** [azerothcore/azerothcore-wotlk#1161]
We need to update our wiki [DB structure](https://github.com/azerothcore/azerothcore-wotlk/wiki/Database-Structure) along with these commits:

- [ ] https://github.com/azerothcore/azerothcore-wotlk/commit/7648858939d9ac1f020955549f748cc973e407a3
- [ ] https://github.com/azerothcore/azerothcore-wotlk/commit/204358bb67ba74fe781ae41ddf79bcaf09aa136e
- [ ] https://github.com/azerothcore/azerothcore-wotlk/commit/468faa3eadc766e59174ca6e43a87892af75e711
- [ ] https://github.com/azerothcore/azerothcore-wotlk/pull/932/files
- [ ] https://github.com/azerothcore/azerothcore-wotlk/pull/1160/files for this one maybe just a general table (not all at once)
- [ ] https://github.com/azerothcore/azerothcore-wotlk/pull/1157

**Comment by FrancescoBorzi** [comment]
@BarbzYHOOL what about re-extracting the whole structure like @Kaev did ?

**Comment by Kaev** [comment]
Feel free to re-extract with https://github.com/Kaev/MySQL-To-CSharp

**Comment by BarbzYHOOL** [comment]
Yeah, I just saw the tool when writing the issue but I had already written it :p
Need windows though

**Comment by Kitzunu** [comment]
Maybe it should be moved to the wiki repo? :P
